frontend/views/containers/contributions/PaymentMethods.vue

```diff
@@ -56,6 +56,7 @@ export default ({
       options: {
         // key to store on state and corresponding translation (if needed)
         bitcoin: 'Bitcoin',
+        lightning: 'Lightning Address',
```

Comment:
Per Slack, how about one of:

- Lightning
- Lightning Addr.
- LN Address

Also, I just realized all of these should probably all be wrapped in `L`, so can you update this PR with that boy-scoutting?